### DISCUSSION TITLE
Aclaración Enunciado E3 - Tarea 4 de la Etapa 3

### DISCUSSION THREAD

**Discussion by calvarezl-cl** [IIC2413/Syllabus-2024-2#285] [General]
Estimadas alumnas y alumnos

En la tarea 4 del enunciado de la etapa 3 se menciona lo siguiente:
Construya un Stored Procedure (1pt) que al final de la carga de las notas del curso a la tabla
termporal ”acta”. Despliegue una **Vista** (1pt) con el acta de notas (en una ventana). El acta de
notas contiene el número de alumno, curso, periodo, nombre del estudiante, nombre del profesor,
la  ́unica nota final que es el resultado de la combinaci ́on de ambas oportunidades (según las reglas
de negocio). Debe validar que existan: el curso, alumno, profesor, que las notas est ́an en el rango
correcto, etc.)

La palabra **Vista** debe cambiarse por **Ventana**

**Comment by SSParzival** [comment]
¿Despliegue una Ventana (en una ventana)?

**Reply by calvarezl-cl** [reply]
Están desarrollando una página web donde, por lo tanto hay un botón que ejecuta la carga de datos y cuando finaliza, llama al procedimiento almacenado que muestra en una ventana (pop up) la información que se solicita.

**Reply by josefa00** [reply]
No entiendo. La pagina web además de tener para hacer las consultas ATC debe tener un boton para desplegar la vista de el acta?
